cocos/core/math/quat.ts

```diff
@@ -401,6 +427,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en The forward direction and the direction of the viewport computing quaternion
```

Comment:
Calculates the quaternion with the up direction and the direction of the viewport